src/index.html

```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="head">
+      <nav class="nav">
+        <a href="./index.html" 
+          class="logo-link" 
+          target="_blank">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+        <ul class="nav_link">
+          <a href="#" class="refer is-active blue" 
+            target="_blank">Apple
+          </a>
+          <a href="#" class="refer is-active" 
```

Comment:
```suggestion
          <a href="#" class="refer" 
```
the is-active class should be only on Apple. Not in each class